lib/Events/ARoomModifiedEvent.php

```diff
@@ -12,6 +12,7 @@
 use OCA\Talk\Room;
 
 abstract class ARoomModifiedEvent extends ARoomEvent {
+	public const PROPERTY_ACTIVE_SINCE = 'activeSince';
```

Comment:
Now the `ActiveSinceModifiedEvent` includes the call flags too. If only the call flags are modified, without touching "active since", a `RoomModifiedEvent` for the `in_call` property is triggered instead.